packages/mail/lib/sendMail.js

```diff
@@ -34,18 +34,10 @@ module.exports = (mail) => {
   // don't send mails if SEND_MAILS is false
   const DEV = NODE_ENV && NODE_ENV !== 'production'
   if (SEND_MAILS === 'false' || (DEV && SEND_MAILS !== 'true')) {
-    console.log('\n\nSEND_MAIL prevented mail from being sent\n(SEND_MAIL == false or NODE_ENV != production and SEND_MAIL != true):\n', mail)
+    logger.log('\n\nSEND_MAIL prevented mail from being sent\n(SEND_MAIL == false or NODE_ENV != production and SEND_MAIL != true):\n', mail)
     return true
   }
 
-  return fetch('https://mandrillapp.com/api/1.0/messages/send.json', {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json'
-    },
-    body: JSON.stringify({
-      key: process.env.MANDRILL_API_KEY,
-      message: mail
-    })
-  })
+  const mandrill = new MandrillInterface({ logger })
+  return mandrill.send(mail)
```

Comment:
An exported, standalone function uses an instantiated object of a file nearby for one command, I don't get it.